src/main/java/org/killbill/billing/client/KillBillHttpClient.java

```diff
@@ -95,6 +95,7 @@
     private final String apiSecret;
     private final AsyncHttpClient httpClient;
     private final ObjectMapper mapper;
+    private final int httpTimeOut;
```

Comment:
My preference would be to rename this `requestTimeout`: it is the timeout both at the HTTP level but also for the request Future (see `executeAndWait`).